acme/setup.py

```diff
@@ -16,6 +16,7 @@
     'pyrfc3339',
     'pytz',
     'requests',
+    'setuptools',  # pkg_resources
```

Comment:
The policy here is to have anything imported inside the source code in the `install_requires` (note, e.g. `mock`, also used in tests only, is included as well). Tests are included in the default distribution, so that anyone can run them on their machine before going live. `testing_extras` or `dev_extras` is additional developer stuff like `pep`, `pylint`, `tox`, etc.